packages/jellyfish-api-core/__tests__/category/governance/governance_container.ts

```diff
@@ -8,7 +8,8 @@ export class GovernanceMasterNodeRegTestContainer extends MasterNodeRegTestConta
   protected getCmd (opts: StartOptions): string[] {
     return [
       ...super.getCmd(opts),
-      '-fortcanningheight=7'
+      '-fortcanningheight=7',
+      '-dummypos=0' // Needed to expire proposals
     ]
   }
```

Comment:
need you to add this code block to prevent pending `1.8.0` conflicts.

```ts
protected getCmd (opts: StartOptions): string[] {
  return [
  // temporary remove -eunospayaheight=7 due to invalid flag
  const cmd = super.getCmd(opts)
  cmd.splice(cmd.indexOf('-eunospayaheight=7'), 1)

  return [
    ...cmd,
    '-fortcanningheight=8',
    '-dummypos=0'
  ]
}
```